cmd/rekor_monitor/main.go

```diff
@@ -44,18 +51,52 @@ const (
 	logInfoFileName          = "logInfo.txt"
 )
 
+// Command-line flags that are parameters to the verifier job
+var (
+	configFilePath  = flag.String("config-file", "", "path to yaml configuration file containing identity monitor settings")
+	configYamlInput = flag.String("config", "", "path to yaml configuration file containing identity monitor settings")
+	once            = flag.Bool("once", true, "whether to run the monitor on a repeated interval or once")
+	serverURL       = flag.String("url", publicRekorServerURL, "URL to the rekor server that is to be monitored")
+	logInfoFile     = flag.String("file", logInfoFileName, "path to the initial log info checkpoint file to be read from")
+	interval        = flag.Duration("interval", 5*time.Minute, "Length of interval between each periodical consistency check")
+	userAgentString = flag.String("user-agent", "", "details to include in the user agent string")
+	runAsMonitor    = flag.Bool("run-as-monitor", false, "Enable this option to expose log consistency check results as Prometheus metrics on /metrics")
```

Comment:
Can we lump this together with `--once`? If `once` is false, then we can assume it's running as a long-lived process and we'll enable prometheus.